checkout_sdk/payments/payments_client.py

```diff
@@ -87,6 +95,13 @@ def get(self, id):
 
         return PaymentProcessed(self._send_http_request('charges/{}'.format(id), HttpMethod.GET))
 
+    def history(self, id):
+        self._log_info('Getting charge history for {}'.format(id))
```

Comment:
Review this if compatibility with Python 2.7 is needed.

Reply:
No, the package supports Python 3.4+